sample-manage-audience/src/main/java/com/linecorp/bot/messagingapidemoapp/controller/BotController.java

```diff
@@ -63,9 +65,22 @@ public String webhook(Model model) throws InterruptedException, ExecutionExcepti
         return "bot/get_webhook";
     }
 
+    @PostMapping("/bot/set_webhook")
+    public String setWebhook(Model model,
+                             @RequestParam("url") URI uri) throws InterruptedException, ExecutionException {
```

Comment:
nit. we can remove `throws InterruptedException, ExecutionException`, isn't it?
```suggestion
                             @RequestParam("url") URI uri) {
```